README.md

```diff
@@ -1,19 +1,46 @@
-
 # Page Checker - Basic web page checking using Selenium
 
 [![Python](https://img.shields.io/badge/Python-3-blue?logo=python&logoColor=white)](https://python.org)
 [![GitHub tag](https://img.shields.io/github/tag/pjdvmalan/page_checker?include_prereleases=&sort=semver)](https://github.com/pjdvmalan/page_checker/releases/)
 [![License - MIT](https://img.shields.io/badge/License-MIT-blue)](#license)
 
-## Installation
+## Raison d'être
```

Comment:
French comment added. Not sure if intentional. Do we need this here or should it be in English? "Reason for being"

Reply:
Me having a moment of fun. Better translation: Reason for existence, i.e. why another Selenium test script?